mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -140,6 +147,92 @@ properties:
       - :PUBLIC_WEBSITE
     immutable: true
     required: true
+  - !ruby/object:Api::Type::NestedObject
+    name: 'documentProcessingConfig'
+    immutable: true
+    description: |
+      Configuration for Document understanding and enrichment.
+    required: false
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        description: |
+          The full resource name of the Document Processing Config. Format:
+          `projects/{project}/locations/{location}/collections/{collection_id}/dataStores/{data_store_id}/documentProcessingConfig`.
+        output: true
```

Comment:
`name` is not marked as "output only" in [the docs](https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1/projects.locations.collections.dataStores#DataStore.DocumentProcessingConfig). However, this seems to be the case.

I've tried creating a new Data Store passing a "documentProcessingConfig.name" that references the one from an existing Data Store, obtaining the following:
* The new Data Store has different (default) documentProcessingConfig params,
* The new Data Store retains the "documentProcessingConfig.name" I specified (same as the existing one)
* The documentProcessingConfig in the existing datastore is left unchanged (so it's different in content from the new one)

Furthermore, setting "output: false" would cause each resource update to send a `"name": null`
param in the request when `name` is not specified, causing an unintended change in the terraform plan.